lua/wire/client/hlzasm/hc_expression.lua

```diff
@@ -45,6 +45,11 @@ function HCOMP:Expression_ExplictIncDec(opcode,label,returnAfter)
     elseif label.Type == "Stack" then
       operationLeaf.Operands[1] = { Stack = label.StackOffset }
     elseif label.Type == "Register" then
+      if self.RegisterIdentities[label.Name] ~= nil then 
```

Comment:
The `if self.RegisterIdentities[label.Name] ~= nil then` is redundant and can be removed.

Reply:
I've removed the ~= nil for each check, let me know if you find anything else